R/doeResponseSurfaceMethodology.R

```diff
@@ -1008,12 +1012,12 @@ doeResponseSurfaceMethodology <- function(jaspResults, dataset, options, ...){
     AnovaTable$dependOn(      options = c("anova","modelTerms", "rsmBlocks",
                                           "rsmResponseVariables",
                                           "rsmVariables"))
-    AnovaTable$addColumnInfo( name = "names",     title = " ")
-    AnovaTable$addColumnInfo( name = "Df",       title = "DF")
-    AnovaTable$addColumnInfo( name = "Sum",      title = "Sum of Squares")
-    AnovaTable$addColumnInfo( name = "Mean",     title = "Mean of Squares")
-    AnovaTable$addColumnInfo( name = "FValue",   title = "F")
-    AnovaTable$addColumnInfo( name = "PValue",   title = "p")
+    AnovaTable$addColumnInfo( name = "names",    title = gettextf(" "))
+    AnovaTable$addColumnInfo( name = "Df",       title = gettextf("DF"))
+    AnovaTable$addColumnInfo( name = "Sum",      title = gettextf("Sum of Squares"))
+    AnovaTable$addColumnInfo( name = "Mean",     title = gettextf("Mean of Squares"))
+    AnovaTable$addColumnInfo( name = "FValue",   title = gettextf("F"))
+    AnovaTable$addColumnInfo( name = "PValue",   title = gettextf("p"))
```

Comment:
these should be `gettext` since you're not interpolating any variables into the string in there.